src/AuthModule/Components/ResetPassword/ResetPassword.module.css

```diff
@@ -0,0 +1,32 @@
+.loader {
```

Comment:
This is another code for the loader, which is not good as I mentioned, as we now have duplicate copies of the same code.